docs_src/microservices/device/services/device-modbus/APIReference.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Device MODBUS - API Reference
+---
+
+# Device MODBUS - API Reference
```

Comment:
```suggestion
# Device Modbus - API Reference
```

Please apply the correct capitals in all the documents

Reply:
updated